Cargo.toml

```diff
@@ -37,18 +37,18 @@ pw = {package= "pipewire", version = "0.5", optional = true}
 serde = {version = "1.0", features = ["derive"]}
 serde_repr = "0.1"
 rand = {version = "0.8", default-features = false}
-zbus = {version = "3.1", features = ["url"]}
+zbus = { version = "3.6", features = ["url"] }
```

Comment:
This changes the style of the file.

Reply:
Yea that was intentional for all the lines touched, the spaces is what most of formatting tools default too as well

Reply:
I mean, if you want to go this route modify it for all crates on a single commit.

Reply:
meh, let's keep it as is then